notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb

```diff
@@ -39,10 +39,10 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "System version: 3.6.7 |Anaconda, Inc.| (default, Oct 23 2018, 19:16:44) \n",
+      "System version: 3.6.10 |Anaconda, Inc.| (default, Mar 25 2020, 23:51:54) \n",
       "[GCC 7.3.0]\n",
-      "Surprise version: 1.0.6\n",
-      "NNI version: 0.5.2.1\n"
+      "Surprise version: 1.1.0\n",
```

Comment:
surprise repo version is 1.0.6

Reply:
we have right now `scikit-surprise>=1.0.6`, so when you install you get the latest, which right now is 1.1.0